src/components/molecules/card/card.html

```diff
@@ -0,0 +1,13 @@
+<div class="card">
+  <p class="two">card-title</p>
+  <div class="block"></div>
+  <figure class="card-img">
+    <img src="../../../img/card-1.jpg" class="card-img-top" alt="..." />
```

Comment:
```suggestion
    <img src="../../../img/card-1.jpg" class="card-img-top" alt="" />
```
> この属性に空文字列を設定すると (alt="")、この画像がコンテンツにおいて重要な箇所ではないことを示し、視覚ブラウザーではない場合は[レンダリング](https://developer.mozilla.org/ja/docs/Glossary/Rendering_engine)を省略することがあります。視覚ブラウザーでは、 alt が空欄で画像の表示に失敗した場合は、壊れた画像のアイコンの表示が省略される場合もあります。

引用: https://developer.mozilla.org/ja/docs/Web/HTML/Element/img#%E5%B1%9E%E6%80%A7

上記のように定義されています。画像が単に装飾用途である場合は空文字列を設定するようにしましょう。